src/vec.rs

```diff
@@ -42,6 +42,43 @@ impl<'s, T> VecAssertions for Spec<'s, Vec<T>> {
     }
 }
 
+impl<'s, T> VecAssertions for Spec<'s, &'s Vec<T>> {
+    /// Asserts that the length of the subject vector is equal to the provided length. The subject
+    /// type must be of `&Vec` with a matching lifetime.
+    ///
+    /// ```rust,ignore
+    /// assert_that(&&vec![1, 2, 3, 4]).has_length(4);
```

Comment:
Can you update the doc tests so it's not being ignored ? This will ensure the doc is always up to date with the actual implementation.

Reply:
Done, thanks for the review.